anta/input_models/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+"""Package related to all ANTA tests input models."""
```

Comment:
test_input_models?

Reply:
Package or module starting with `test` could be confusing with pytest.